### PR TITLE
Draconic language, for lizards

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Restrictions on Lizardpeople using their native language on the
station have been lifted by Centcom, in order to maximise worker
productivity. The language's key is ",o".
/:cl:

- The language is added as part of the job equipping code, changing your
species to lizard via xenobio will not magically teach you the language.
- Normal human tongues are capable of speaking Draconic.
- pAIs with translator and Librarians are capable of speaking and
understanding Draconic.

The design lead has instructed me to grant lizardpeople their own
language, in order to increase paranoia in the crew. Also, it'll be
funny to see how much people overreact to this.

### CODE REVIEW

**Comment by hornygranny** [comment]
delet this

**Comment by dannno** [comment]
if this is added and I see a fucking ligger talking in his mud tongue I will spread his essence across the walls

delet this

**Comment by Cyberboss** [comment]
@hornygranny Crawls out of the woodwork to say we've gone too far

**Comment by juanototo2** [comment]
**delet this**

**Comment by Durkel** [comment]
delet this

**Comment by lzimann** [comment]
This is stupid, giving languages to roundstart races is stupid.

**Comment by GunHog** [comment]
Engaging salt collectors. This time, it is YOU who is generating the drama, @coiax!

**Comment by juanototo2** [comment]
### GOOF WAS RIGHT SLIPPERY SLOPE HE CAN'T BE PROVEN WRONG

**Comment by KorPhaeron** [comment]
>game allegedly about misinformation and paranoia
>coiax adds something that lets us control that flow of information and cause suspicion between players
>people decide to yell about furries

Sad!

**Comment by Incoming5643** [comment]
Do not call it Unathi our lizards aren't those impure souls

**Comment by dannno** [comment]
change it from unathi tbh

**Comment by dannno** [comment]
call it "mud speak" or "mud tongue"

**Comment by PKPenguin321** [comment]
Config?

**Comment by KorPhaeron** [comment]
Yes we have a config for lizards

**Comment by Tacolizard** [comment]
Changing your species to lizard should teach you the language.

**Comment by Kromgar** [comment]
@hornygranny HOLY SHIT WHAT

**Comment by coiax** [comment]
> unsubscribing from my own PR

**Comment by Armhulen** [comment]
>unathi

File: code/modules/language/unathi.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	name = "Unathi"
+	desc = "The common language of lizard-people, composed of sibilant hisses and rattles."
+	speech_verb = "hisses"
+	ask_verb = "hisses"
+	exclaim_verb = "roars"
+	key = "o"
+	flags = TONGUELESS_SPEECH
+	space_chance = 40
+	syllables = list(
+		"za", "az", "ze", "ez", "zi", "iz", "zo", "oz", "zu", "uz", "zs", "sz",
```
Comment (octareenroon91) [inline code/modules/language/unathi.dm]: First time I was opening this code, I'd hoped to see `"s", "S",` and nothing but.

**Comment by GunHog** [comment]
>> unsubscribing from my own PR

Oh no, @coiax! You must embrace the drama, or be consumed by it!

**Comment by MMMiracles** [comment]
people getting upset over actually utilizing the language system are dumb, tbh

**Comment by InsaneHyena** [comment]
GOOFBALL WAS RIGHT AGAIN

**Comment by PKPenguin321** [comment]
>Yes we have a config for lizards

mfw this PR will give me an excuse to disable lizards
my face when

**Comment by Lumipharon** [comment]
Neat.

tunnel snake slangtalk when?

**Comment by BeeSting12** [comment]
only if lizards are valid for speaking it tbh

**Comment by octareenroon91** [comment]
>only if lizards are valid for speaking it tbh
Silicons have had open season for a long time.

**Comment by blobbernaut** [comment]
Anything that makes lizards trigger people more. +1.

**Comment by KorPhaeron** [comment]
No actual arguments yet, just random angst

**Comment by MMMiracles** [comment]
the only argument I could see is it supporting cliques/metagroups, even then that can be dealt with ICly.

**Comment by optimumtact** [comment]
Adding a lizard language is only going to further the tension between lizards and humans @KorPhaeron I think this is a big mistake and does not fit with the playerbase.

What happened to the theory that non human races wouldn't have significant differences that caused imbalances? if I remember Kor you were quite a big pusher of that viewpoint.

**Comment by Repukan** [comment]
>Unironically wanting to bring the bullshit that perpetuates meta cliques in Bay and Paradise

Goof was right

File: code/modules/language/unathi.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/datum/language/unathi
+	name = "Unathi"
```
Comment (optimumtact) [inline code/modules/language/unathi.dm]: can we not steal all our lore from bay please

**Comment by KorPhaeron** [comment]
Plasmamen got added with pretty significant balance differences (flash immunity in rev, for one) and our game didnt collapse.

I think tension from sources other than the antagonists is good to keep the game fresh and prevent it from just becoming TDM

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (octareenroon91) [inline code/modules/language/unathi.dm]: "Serpens"
"Forktongue"

Comment (optimumtact) [inline code/modules/language/unathi.dm]: "scalescreech"

**Comment by optimumtact** [comment]
So your plan to prevent the game from becoming a team deathmatch is to further delineate the crew into teams based on their racial/language attributes?

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (FantasticFwoosh) [inline code/modules/language/unathi.dm]: Ash-Pidgin, basically a mish mash of words

Comment (dannno) [inline code/modules/language/unathi.dm]: mud tongue

**Comment by Jazaen** [comment]
Plus, not giving AI an ability to understand it.

**Comment by lzimann** [comment]
What makes you think this do anything to prevent "TDM", when in reality this adds more kill first ask later scenarios.

**Comment by Militaires** [comment]
lizards were originally added with the intent to keep them as a re-textured or even downgrade from humans (no humanity aka silicon protection, digi legs no shoes).

this breaks that balance and effectively gives all lizards their own channel. in addition to the fact that it will inevitably perpetuate metagangs.

>but this will create more tension1111111111111111111111111111111111111111111111

no it wont, people will start to go lizard JUST to get the language and you cant disagree, people who dont play lizards wont give a shit that humans speak that because the people who WOULD care are the peopl who would change race for it.

**Comment by Militaires** [comment]
in my opinion, we should remove mutant races if you're actually aiming for a less TDM gameplay.

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (Armhulen) [inline code/modules/language/unathi.dm]: shittalking

**Comment by Iamgoofball** [comment]
# SLIPPERY SLOOOOOOOOOOPE

**Comment by Militaires** [comment]
- >lizards are added as a race alternative as a strict reskin downgrade for humans 

- > lizards given extended customization 

- > lizards given digitgrade legs

- > lizards given their own separate language

let's not stop here and try to **FORESEE** into the future

- > lizards given their own species strengths

- > lizards made human under asimov 

- > lizards split from species and made their own subtype of carbon

here's what i have to say about this "phenomenon"

the game never had any focus and it will never will, but there are certain things we try to avoid, things that take away from the original experience we signed up here, be it good sportsmanship and interesting, complex, or deep gameplay or roleplay.

obviously, this experience is starting to "fade away", and discussion on how to reverse this effect is undergoing, but we can not talk about how tg has turned from a deep, rich environment to a "catch the valids: the game" while we continue to go down this path of further PROMOTING that type of gameplay. 

**people often laugh at posts like this and drop a comment or two about how we're "overreacting", and that we're idiots for caring about it at all** (hell! it's in the OP as I write this!). but they don't understand that it's not about lizards having snowflake legs or not, it's that if you keep breaking up the crew into all sorts of factions, give **OBVIOUS advantages** that powergamers will pursue, (this) promoting even more "become the strongest on the station" gameplay.

maybe im going on a rant, but it's not even about this lone individual PR anymore, it's about how the game has become over time and how it has a "lifespan" that we may very well be shortening. **that doesn't matter though, because I'm just overreacting over lizards getting more snowflake, right?**

**Comment by MMMiracles** [comment]
I don't see how mostly cosmetic changes (digitgrade also hindering lizards) is going to lead to them gaining full ASIMOV protection and and somehow become the only non-human species with genuine strengths (plasmamemes being the only species with a 'strength' in that they can become a flaming inferno and die trying to set others on fire).

but DAE SLIPPERY SLOPE BAD CANTINASTATION GO HOME

**Comment by dannno** [comment]
lmao nigga u wrote a book wtf

**Comment by Militaires** [comment]
@MMMiracles i mean this is a huge buff in and of itself so that thought wont seem so alien afterwards

@dannno l# **ol that's not a book ill show u what a book is after this post**

**Comment by Cyberboss** [comment]
Admins please make liggers who use the language valid

**Comment by optimumtact** [comment]
lad you are having a giggle
I'd like a dose of whatever you're taking

**Comment by Militaires** [comment]
It has come to my attention that recent changes in the tgstation gaming environment have led to this ultimate catastrophe where we are heading down towards the underground real big daddy basement of shitty gameplay, not only did we lose the big boy big bucks big girls and the company is all falling apart you dumbfucks you cant save this shitty investment, but we're also boring as fuck i might as well play first person fucking pooper in my asshole you wanker i bet you're an aussie autist danno

what does this mean? liggers are fucking gene trash that was let in by kor "i jack off and draw with my hand and call it a gf" phaeron cause he's a massive cuck and dumbfuck that thinks a shitty ligger tail between his legs counts as a dick since his dwayne johnson is so fucking tiny the drone in his avatar has a bigger peepee, with that being said i'd like to tell you that our gameplay has become a shitty fucking shithole cesspool like the holes full of bodies and shit in outlast 2 when the fuck did you go wrong, there's absolutely nothing to do in the game that takes basic mental thought or interaction and autists like kevinz and oldman robustin ruin this asshole for everyone.

"hahahahaha ur shitposting" eat my ass you cancerous fuck, if you as much as whisper your autist mating call ill shove a fork up your ass and hang you upside down like a fucking nigger after a KKK facereveal you ass wipe, the only joy you _deserve is joy la_ne because being shot is the only thing you seem to be good at, sometimes you get shot with cum since you're such a huge fucking faggot and sometimes **you get shit on for being the aforementioned faggot**

the meth is the only thing keeping me sane and the ever so often thought that maybe youll turn around this car crash, actually train crash, crashing with a s# choolbus full of autistic children and fix it and make tgstation a good place where i can have fun and be swordfish hackerman and have something res## embling a normal fucking CONVERSATION, **delete your fucing existence**

**you're a real fucking shithee**l whose face needs to be kicked on the curb until the autism spills out of it @dannno, the moment you tried to mess wit hthe big fucking man himself i hit you like a dose of that vaccine made you even more braindead and autistic you fucking ADHD chimp, i hope you suck on oranges' diseased tit until he stops giving you that sweet sweet shitmilk and you wither and die like a dishgusting fucing aphid on a stalk you piece of shit PARASITE, not even a jew sucks up and wastes oxygen as much as you

the moment you try to write something it spills onto the front page like shit out of an asshole, if i had the birth of AIDS on tape it'd be your inbred parents' sextape cocksucker, bleach drnks itself the moment it sees you because it cant comprehend how a human being could look like a fucking cow so much, if you w**ent to india you'd be mistaken for a designated shitting street you fat fuck, lose some fucking weight and get off the cheeto dust cause it's mixing with the meth you take daily and your crusty fucking lungs are starting to pop out in your manboobs you landwhale.

a community full of autists playing a special game for special people trinyg to add special features for their special charact**ers? you're maing this a fucking shithole and im not staying to watch this orgy happen fuckboys.**
al people trinyg to add special features for their special charact*rs? you're maing this a fucking shithole and im not staying to watch this orgy happen fuckboys.**

**Comment by Militaires** [comment]
for clarifiaction i only posted that so that oranges replies and i post the paragraph but he deleted it so lmao

**Comment by Militaires** [comment]
@optimumtact did you fucking delete my book i spent effort writing that dude

**Comment by Hatfish** [comment]
It'd actually be pretty neat if sec could arrest employees for speaking a non-standard language on the station, if this gets through someone should go add that to Space Law.

**Comment by optimumtact** [comment]
@Hatfish space law is an ic guideline only so has no bearing on the actual rules

**Comment by Militaires** [comment]
# ok ill stop shitposting

>>>>>this is not ok because people will powergame with it every round and always pick ligger to be able to speak secret club language, make it use tongues????

**Comment by Bawhoppen** [comment]
this will be such a fun change

**Comment by obscolene** [comment]
nice change, i love it!!!! xD can you also add lizards 🅱️eing able to decapitate people unarmed?? thanks!!! :3c

**Comment by KorPhaeron** [comment]
@optimumtact

Obviously it isn't going to magically solve TDM on its own but I think it adds flavour and possible motives/subertfuge rather than just piling more reskinned guns and traitor murder items into the game. 

@Militaires

Anyone who tries to powergame as lizard will quickly discover that the AI can and will kick their shit in if they so much as defend themselves from a human

**Comment by KorPhaeron** [comment]
Also maybe nostalgia has poisoned my brain but I actually miss the obnoxious lizard cliques and their drama, it was fun to get mad at and find ways to work with/against/around them.

**Comment by sawrge** [comment]
abime

**Comment by GunHog** [comment]
Any chance of allowing silicons to understand it? It is a roundstart language which NT allows, not something special like Swarmer or such.

**Comment by KorPhaeron** [comment]
Silicons arent programmed to care about lizards sorry

**Comment by calzilla1** [comment]
Ok, but let humans speak a special snowflake language (russian?)

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (octareenroon91) [inline code/modules/language/unathi.dm]: "Herp Chrip"
"Lig Jive"

**Comment by Sligneris** [comment]
> Restrictions on Lizardpeople using their native language on the station have been lifted by Centcom, in order to maximise worker productivity.

>\> Allowing lizards to speak metagang language
>\> This increases productivity somehow

**Comment by optimumtact** [comment]
Well we find that crew quality of life rating skyrocket after they murder all the lizards on station and letting them speak their own language seems to facilitate this to a greater degree Crewman Sligneris

**Comment by Bawhoppen** [comment]
they need to be ABLE to speak it, but not be allowed by space law

**Comment by demonfiren** [comment]
excellent feature

merge posthaste

**Comment by Sligneris** [comment]
filthy lizard

**Comment by XDTM** [comment]
The librarian is gonna be very importamt in interspecies relationships. Maybe the lawyer should know lizard as well, and janitors too, regardless of race.

**Comment by coiax** [comment]
LIBRARIAN TO BRIG! GOT A LIZARD GANG!

**Comment by lzimann** [comment]
Those things all sound great in your heads, but it's never going to actually happen.

**Comment by Aloraydrel** [comment]
Always funny to see people doomsaying how the boogeyman powergamer will play lizard so he can ooc/ic talk to his fabled metabuddies who ruin the station

**Comment by Sligneris** [comment]
About as funny as you people shrugging off adverse effects of their reckless ideas.

**Comment by AnturK** [comment]
Not really a fan, doubt this will create any interesting situation that whisper already does not cover. And i'm not sure trying to make lizards more coherent group is good idea.

Also kor stop using lack of paranoia as a excuse for literally any changes.

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (AnturK) [inline code/modules/language/unathi.dm]: Yeah, this gets a 🚫 from me.

**Comment by Dorsisdwarf** [comment]
Actually I like this but only if silicons get it, so the AI can rat on the lizzies to sec.

Also imagine the hilarity if the librarian tells sec the lizards on the radio are revs

**Comment by Repukan** [comment]
>Those things all sound great in your head, but it's never going to actually happen

**Comment by coiax** [comment]
It would make sense for silicons to have a variety of common languages in their database. I've thought of how to do it in a non terrible way too.

**Comment by coiax** [comment]
But I would obviously need to talk to the lore master about it.

**Comment by GunHog** [comment]
> But I would obviously need to talk to the lore master about it.

Because that totally applies when the librarian and a sentient pocket computer knows how to talk like a sapient robot, repair robot, robot termite, clockwork robot, and a xeno! Besides, since they employ lizards, NT probably has the language on file.

**Comment by coiax** [comment]
Touche.

**Comment by TrustyGun** [comment]
There's a lore master?

**Comment by Cyberboss** [comment]
https://github.com/tgstation/tgstation/blob/master/config/game_options.txt#L397

**Comment by duncathan** [comment]
>people disagreeing with the design lead

wew

**Comment by KorPhaeron** [comment]
I am not infallible

**Comment by Bawhoppen** [comment]
the only reason I can why people wouldn't support this is if they are nonironically lizard haters

**Comment by coiax** [comment]
@Bawhoppen but then powergamers like @kevinz000 are going to play lizards in order to take advantage of the MAD LANGUAGE POWERS!!!!

**Comment by GunHog** [comment]
But Kev cannot powergame as HoS if he is a lizard! And being a lizard Sec officer is a really bad idea if you upset the AI!

**Comment by lzimann** [comment]
First of all, Kor, I like you, but please, you gotta stop using the excuse of the game being "too TDM" or how some features will help bring the game to a "more RP state" when it won't change anything in this regard, it won't magically make people RP because now they can speak in a different language, heck, as I said before, it will only help propagate even more the kill first ask later situations. What you want to achieve is already achievable with whispers, no extra language is required, and I ask you, how many times does that even happen? No conflict is generated in these situations, at least I will not happen with us.

The lizard metagang was awful. If you happened to have one of them as target you were fucked, because they would be together all the time, making you go against 4 instead of 1(and if you got into a fight with one, you would have 3 others that would take his side regardless of what happened).

**Comment by Repukan** [comment]
Lzimann x Korphaeron OTP

**Comment by Bawhoppen** [comment]
why do people constantly theorycraft about imaginary lizard metagangs I don't get it

**Comment by Repukan** [comment]
>Happened before
>Imaginary
Pick one

**Comment by Bawhoppen** [comment]
you're only supposed to say that when someone contradicts themselves

**Comment by Cyberboss** [comment]
Lizard metagangs have happened before

**Comment by Iamgoofball** [comment]
i've personally arrested lizard metagang members and had their metabuddies harass the brig for it

**Comment by TheColdTurtle** [comment]
Just execute them goof, not like the Ai will care

**Comment by Militaires** [comment]
how can you guys (namely kor) be like YES, THIS WILL INCREASE TENSION!!! AHA!!! GENIUS!!!

when you clearly know the result of this like fug man it's as clear as the sun and that all it will do is just further divide the crew

**Comment by XDTM** [comment]
Why aren't lizard metagangs happening now, they don't need a private language to metagang since it's happened before

**Comment by TheColdTurtle** [comment]
But wouldn't dividing the crew create tension?

**Comment by Sligneris** [comment]
People tried to say this Militaires, it's been shrugged off as "just random angst"

**Comment by Aloraydrel** [comment]
Literally nothing will happen besides your fear of people talking and becoming friends overtime

**Comment by KorPhaeron** [comment]
@iamgoofball

Same and then I kill all of them

**Comment by KorPhaeron** [comment]
@Militaires

How will dividing people not create tension??

**Comment by KorPhaeron** [comment]
@Cyberboss

And if they happen again to the point where they ruin gameplay they'll be kicked out, it is not like admins cant read what they say to one another

**Comment by KorPhaeron** [comment]
@lzimann

A design goal isn't an excuse you meme. You're right this probably will have almost zero impact on how people play the game though.

**Comment by lzimann** [comment]
It is an excuse when you try to use the design goal on something that would fail to achieve that though.

**Comment by Sligneris** [comment]
> Literally nothing will happen besides your fear of people talking and becoming friends overtime

... or only get more and more annoyed as lizards refuse to talk normally, and only talk their own language, refusing to interact with anyone outside their species. Then, when lizards who speak only their language get what's coming to them, angry admins come to their defense, resulting in obnoxious players becoming essentially untouchable.

I've played on a servers with species languages. I know how they work. They're *nothing* but a middle finger to those who can't understand them, they serve *no* other purpose but annoy people outside a special [species] circle.

**Comment by KorPhaeron** [comment]
Sligneris> as for roleplay - I don't like the way any RP servers handle it. in fact, I despise it. I like the idea of an immersive environment, a world to be a part of... but it's not something I seek from ss13 as it is now

**Comment by coiax** [comment]
I'm not sure if people sitting in the bar talking to each other rather than going out validing things is a bad thing.

**Comment by Sligneris** [comment]
I mean, then there's plenty of servers that are already little but sitting in the bar and talking to each other. You might give them a try and see if they're any more fun for you.

**Comment by lzimann** [comment]
Technically, yes, it is a bad thing. It doesn't affect much the game as it is now, but you would have people that are not doing what would be expected them to do(which is the very least, their job).

Imagine a scenario where everyone would just go to bar. So instead of people being antags, creating conflicts(after all, it was said multiple times in this PR that it is the goal of the game) you would have a snoozefest of people talking to each other in-character.

**Comment by coiax** [comment]
We do have a whole bunch of jobs called Service jobs, who are supposed to supply the crew with food, drinks and entertainment, and it's a bad thing that they're being used?

Or should the Barkeeper be renamed to Shotgun Assistant?

**Comment by Militaires** [comment]
@korphaeron it'll create tension, but if your idea of tension is RACE WAR NOWW11111111 autism every round or lizards metaganging then yes you got your tension

**Comment by MMMiracles** [comment]
sounds like a pretty good way to weed out the shitty people tbh

**Comment by Militaires** [comment]
"but we'll ban them" good luck bannin' most of our demographic lol

**Comment by XDTM** [comment]
i think the problem is more with the "lol racism is funny" group i think

**Comment by XDTM** [comment]
also not the only one

**Comment by KorPhaeron** [comment]
I'm no longer convinced this is a good idea I don't think it will actually impact rounds at all in ways that whispering or pda messages couldnt

**Comment by Militaires** [comment]
**no problem guys i fixed the rogue kor and reset its laws using my meme techniques ur welcome**

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

Alright I've been convinced it probably won't actually improve paranoia or anything but I also don't think this will lead to the ERPers coming back and destroying the server. I acknowledge I actually have no idea what this would do, so I think it'd be worth testing though since we don't really have anything comparable and this can tell us if more languages are worth pursuing or not.

Last chance to talk me down etc

**Comment by lzimann** [comment]
I already gave my opinion on this. I don't think adding a roundstart language that won't be accessible by everyone is a good idea, any kind of benefit to a specific race should be disencouraged, they should be all cosmetic.

**Comment by cedarbridge** [comment]
The concept that anything that isn't a bog standard human should be so laden with downsides to discourage their play is retarded from a design angle.  Non-human races already have various IC consequences from the autistic playerbase and the scilicons have started hopping in because asimov doesn't provide protections for non-humans (as it should remain.)  Adding different races with different functions upsides and downsides should be balanced around the human model as the base, as it is in most RP settings, and it should go from there.  This whole autistic "furries/scalies reeeeee" thing is juvenile.  At worst this encourages players to use the already criminally underused Librarian and pAI roles for something other than memes and suicide fodder.  pAIs are currently just really shitty doorjacks so giving them translation jobs is super useful but only when there are languages to translate.

tl:dr: Good change, merge posthaste, more similar in the future. Also add human secondary languages (gutter/tradeband/etc).  Stop trying to make non-human races humans with tails and only downsides.

**Comment by kevinz000** [comment]
pais are only really shitty doorjacks because i'm going to get my face put on a stake if i tried to make them more
oh yeah i should take out their mobility function and make them sentient tablets and give them more shit >_<

**Comment by Pascal125** [comment]
Have you lads stopped to consider that maybe the problem here isn't this fictional "Lizard Metagang" but the ones who go out of their way to kill people just for playing a different race? It's just a different language. It doesn't have to turn into a political debate about all these buzzwords thrown around.

**Comment by optimumtact** [comment]
@cedarbridge that's a fairly absurd position to take there, nobody is suggesting "that anything that isn't a bog standard human should be so laden with downsides to discourage their play is retarded from a design angle."

The question is actually, how many upsides/distinctions do we want to give non human races and do they contribute to the culture and gameplay in any meaningful way

**Comment by PKPenguin321** [comment]
Can we close this now

**Comment by KorPhaeron** [comment]
>do they contribute to the culture and gameplay in any meaningful way

Only one way to find out. There is not really any precedent for this

**Comment by Jordie0608** [comment]
Testmerge this if you want to trial it; it shouldn't be merged if you're apprehensive about how it'll even go.

**Comment by KorPhaeron** [comment]
@Jordie0608 

It needs long term testing, the first week or two at least I expect people to throw tantrums about this.

**Comment by Bawhoppen** [comment]
I honestly can't fathom why anyone would be against this

**Comment by Aloraydrel** [comment]
lizard metagangs will rule the server again

**Comment by cedarbridge** [comment]
> I honestly can't fathom why anyone would be against this

Because there's this prevailing belief among some that any race in the game should be a cosmetic reskin of a human but with none of the roles or AI protections given to humans.  For some reason.

**Comment by lzimann** [comment]
There are many things I can't imagine, yet they happen.

**Comment by Bawhoppen** [comment]
literally all I see is people unironically being racist against 2d lizards

File: code/modules/language/unathi.dm (hunk 2 above)
Comment (GunHog) [inline code/modules/language/unathi.dm]: How about "Lizan"?

**Comment by GunHog** [comment]
If I ever wanted to do a study on bigotry, doing it artificially using pretend races so that no one is actually offended would be the way to go. I wonder what someone researching this type of psychology would say about this and the whole 'Normies vs Snowflakes' controversy. Furthermore, I continue to wonder what divides a community so strongly. At its core, behind all the metagangs and circles, I wonder if this is a bit like typical high-school dynamics where one or more groups class others as inferior, forming a sort of hierarchy. I could be overreaching here; It might be little different than _Pro Skub and Anti Skub_ for all I know!

Anyway, about the actual feature, it would be nice if the name "Unathi" were changed to something else so as not to be copying Baystation, and also allowing silicons understand it. That change alone would prevent it being as strong of an advantage when used over the radio.

**Comment by Repukan** [comment]
>Had old problems with lizard metagangs
>Even today some of the taint persists
>It's proven by looking at other servers that giving other species languages brings players that form cliques or refuse to communicate in sol common
>Instead of looking at this as a con we're just gonna ignore it because the people against it are just probably bigots or repressed high school students

Jogs the nog

**Comment by octareenroon91** [comment]
> looking at this as a con

Who's conning whom, though? I'm holding out for some kind of plot twist here.

**Comment by PKPenguin321** [comment]
Con as in the opposite of a pro

**Comment by GunHog** [comment]
I believe he was making a joke!

**Comment by Repukan** [comment]
Aren't jokes supposed to be funny?

**Comment by GunHog** [comment]
They need only be funny to either the person making it or at most his intended recipient. Have you ever met a clown? :laughing:

**Comment by sawrge** [comment]
If I can't make friends, no one can!

**Comment by KorPhaeron** [comment]
@Repukan

But we has even worse human metagangs and neither group needed languages to form because they all used skype or discord anyway

**Comment by Sligneris** [comment]
@coiax

\>\> Baystation12

@sawrge @Bawhoppen

\>\> Paradise Station

Problem solved, everyone gets the kind of server they want.

**Comment by Bawhoppen** [comment]
@Sligneris Actually I think you're the one who doesn't below at tg:
https://hippiestation.com/

**Comment by demonfiren** [comment]
just merge this and listen to the sound of all the doomsayers shutting up

but yes, janitor should totally speak lizard regardless of race

**Comment by KorPhaeron** [comment]
@demonfiren

The people who were against map rotation decided to prove it was bad by griefing on maps they didnt like and then complaining that proved the maps led to bad rounds, I doubt people are just going to shrug and move on from this quickly if at all

**Comment by coiax** [comment]
> I literally just got told to "go to bay".

**Comment by optimumtact** [comment]
yeah the only question you should be asked is "why haven't you left yet?"

**Review by Spess-Rando — APPROVED** [review]

**Comment by Sligneris** [comment]
@coiax 
At least now you're not literally porting "Unathi". That's an improvement.

@Bawhoppen 
Oh yes, surely wanting faithful SS13 experience untainted by autistic cliquey bullshit, is the same as wanting to hear unending /pol/-style rants about jews and blacks.

Part of the reason I directed you to Paradise and not Bay is because you're unironically make argument against 'fursecution', now with lizards.

**Comment by jughu** [comment]
>adds langauge system
People be like oh cool
>adds languages 
DELET THIS

**Comment by Pascal125** [comment]
@KorPhaeron Then use it as an excuse to get rid of people who cause trouble. Just like you should've with the map rotation. It ain't gonna fix itself, y'know?

**Comment by dannno** [comment]
pfffhahahaha well there it is boys, it's approved

seriously though "draconic" is even worse than unathi

**Comment by GunHog** [comment]
Let Ash Drakes and Lesser Ash Drakes (miners who get the spell power) speak it too

**Comment by Sligneris** [comment]
@danno
Yeah, I meant it as a 👍 for the change to "draconic", but sadly Github treats it as general approval.

Still 👎 on the PR, it's just a little less awful now.

**Comment by demonfiren** [comment]
just meme and call it space hebrew

**Comment by calzilla1** [comment]
>adds language system
>people be like "oh cool"
>give a special snowflake language to human reskins giveing them a massive edge
>DELET THIS
If you're doing this add russian for postarity and to I N C R E A S E T H E M E M E S

**Comment by KorPhaeron** [comment]
>massive edge

?

**Comment by GunHog** [comment]
It is exaggeration, Kor, much in the way you often do so with the words "infinite" and "unlimited". There is an advantage in having a language that others cannot use, being partially secure communication, If care is taken not to speak over the radio so as to avoid letting the librarian or a pAI hear you. There are superior methods of course (PDA, non-standard radio freqs, antag comms), leaving this one to be more of an annoyance than a tactical advantage.

**Comment by KorPhaeron** [comment]
I think lizards are the best starting point for expanding languages because

-Part of the crew

-They are a minority, but a sizeable enough one that the language might actually get used. Not enough people play fly or plasmaman

-If we added it to subsets of humans, it could be used as a way to foil disguises (you don't know assistant speak! Traitor!). You can't disguise as lizards anyway because of the stupid tails and hissing.

I think this should be merged for a ~month, and we can then evaluate if we want to revert it, or expand languages to other groups, etc

**Comment by demonfiren** [comment]
the actual madman

**Comment by coiax** [comment]
Beat that @GunHog. Maybe you could try making some sort of Security Catlizard job, that has its own gun.

**Comment by GunHog** [comment]
Oh...my...impressive, Coiax....

**Comment by Sarcalogo** [comment]
\>This shit was merged.

Jesus Christ, you people really are retards aren't you. You didn't even add a human only language that only humans can understand. Seriously niggers.

**Comment by lzimann** [comment]
#26856

**Comment by Iamgoofball** [comment]
its ok sarcalogo i got u

**Comment by KorPhaeron** [comment]
Round 1 of watching this: A a couple lizards shouted on the radio, some people shouted that the language was dumb back at them, one lizard read WGW in draconic and got gibbed, then everyone went back to communicating normally. 

Pretty underwhelming so far.

**Comment by Sarcalogo** [comment]
\>metaclubs form in one round
Yeah

**Comment by Bawhoppen** [comment]
NaCl

**Comment by Dorsisdwarf** [comment]
>Ported to citadel

Well that's a bad feature right there

**Comment by optimumtact** [comment]
I dont' know if you're a bit damaged in the brain but both citadel and paradise are auto mirroring our pr's, a fact which should be imediately obvious upon cursory inspection

**Comment by Sligneris** [comment]
Citadel and Hippie, not Citadel and Paradise.

**Comment by optimumtact** [comment]
they're all the same

**Comment by TheDreamweaver** [comment]
Hey guys wanted to hop in to add my important words of: who the fuck cares its a game please just have fun thanks bye

**Comment by Militaires** [comment]
@coiax @GunHog 

if you jabronis think you're good at stirring shitstorms you couldnt even compete with the KING

**Comment by GunHog** [comment]
O SNAP
